openquake/hazardlib/tests/geo/geodetic_test.py

```diff
@@ -390,6 +390,21 @@ def test_zero_distance(self):
         self.assertTrue(numpy.allclose(lats, expected_lats))
         self.assertTrue(numpy.allclose(depths, expected_depths))
 
+    def test_input_as_int(self):
+        lons, lats, depths = geodetic.npoints_towards(
+            lon=0, lat=0, depth=0, azimuth=0,
+            hdist=0, vdist=5, npoints=7
+        )
+        expected_lons = [0, 0, 0, 0, 0, 0, 0]
+        expected_lats = [0, 0, 0, 0, 0, 0, 0]
+        expected_depths = [0, 0.83333, 1.66667, 2.5, 3.33333, 4.16667, 5]
+        self.assertTrue(numpy.allclose(lons, expected_lons))
+        self.assertTrue(numpy.allclose(lats, expected_lats))
```

Comment:
Please use `numpy.testing.assert_almost_equal`, `assertTrue` gives a terrible error message when the test breaks.